test_runner/src/test/kotlin/ftl/fixtures/flank.local.yml

```diff
@@ -10,6 +10,7 @@ gcloud:
   async: true
   test-targets:
   - class com.example.app.ExampleUiTest#testPasses
+  - $FROM_ENV
```

Comment:
maybe it should be placed in a separate file to avoid having empty test-targets for anybody who does not set this `FROM_ENV` env?

Reply:
sure, why not 👍🏻

Reply:
On second thought, I think it should stay as it is now. Yeah, in most cases it will produce nothing (we will have only `ExampleUiTest#testPasses`) but that would also verify if flank can skip empty variables without any problem. 
Let me know what do you think